BootloaderCommonPkg/Library/IppCryptoFipsLib/IppCryptoFipsLib.inf

```diff
@@ -0,0 +1,115 @@
+## @file
+#
+#  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = IppCryptoFipsLib
```

Comment:
Can we remove FIPS from the naming? and module name as IppCryptoLib2?
What do you think?

Reply:
@gdong1 @tsaikevin do you have an opinion on this?

Reply:
If having multiple library instances, often the difference is shown at the end of the library following EDK2 naming.
Using BootloaderCommonPkg/Library/IppCryptoLib2/IppCryptoLib2.inf or BootloaderCommonPkg/Library/IppCryptoLibFips/IppCryptoLibFips.inf. Maybe Subash think new library changes has more than FIPS so he suggest to use IppCryptoLib2.

Reply:
Yes, our go is to use IPP crypto from source directly. FIPS is the compliance which is getting achieved with this.

Reply:
@lsubash @gdong1 library has been renamed to IppCrypto2Lib.

Reply:
Thanks for the update. it looks good to me.